src/hotspot/cpu/riscv/vm_version_riscv.cpp

```diff
@@ -186,6 +186,10 @@ void VM_Version::initialize() {
     FLAG_SET_DEFAULT(UseMD5Intrinsics, true);
   }
 
+  if (UseRVV && FLAG_IS_DEFAULT(UseChaCha20Intrinsics)) {
+    FLAG_SET_DEFAULT(UseChaCha20Intrinsics, true);
+  }
```

Comment:
Just below this we may set RVV to false.
I would put this just above "#ifdef COMPILER2" or so.

Reply:
:+1: https://github.com/openjdk/jdk/pull/15899/files#diff-7b173d6e5834de13749c8333192fef5a874628a67b90a5d8d06235d507542ac4R255 seems like the ideal place.

Reply:
Thanks Robbin, Fixed.